web/src/pages/Service/service.ts

```diff
@@ -44,7 +47,7 @@ export const create = (data: ServiceModule.Entity) =>
 export const update = (serviceId: string, data: ServiceModule.Entity) =>
   request(`/services/${serviceId}`, {
     method: 'PUT',
-    data,
+    data: omit(data, DELETE_FIELDS),
```

Comment:
I found that every `update`  method omited `DELETE_FIELDS`, it would be better to omit these field when render the editor, instead of before submit, which may induce users that we allow modification of these fields, but actually not.

Reply:
Agree with you. This will give the user a better experience, I'll update it later.

Reply:
updated. @liuxiran